docs/users.md

```diff
@@ -0,0 +1,86 @@
+# Create a Database User #
+
+You can create a MongoDB database user to authenticate to your MongoDB resource using SCRAM. First, [create a Kubernetes secret](#create-a-user-secret) for the new user's password. Then, [modify and apply the MongoDB CRD](#modify-the-mongodb-crd).
+
```

Comment:
Question about behavior here: When you create one SCRAM user, you will no longer be able to perform any actions unauthenticated, correct? I think we can be a little clearer about this here.

Reply:
There are a handful of commands you can run without being authenticated, like the native commands (like `listFiles()` ) and a few database commands. I tried `db.runCommand({connectionStatus : 1})` and `isMaster`, which both worked while unauthenticated. But I can't find a definitive list of them, and I don't think the ones that work do very much. Let me know if you want me to elaborate more in the topic. And maybe @chatton  has an opinion :)

Reply:
I think it should be clear enough to link to the SCRAM docs like we do here, as everything written there will apply, I don't think we need to clarify any more about this, I don't feel too strongly about this though.